Updating Legacy JSON.md

```diff
@@ -0,0 +1,159 @@
+# Updating Legacy JSON
+
+Use the `home` key to get to the top. 
+
+- [Introduction](#introduction)
+- [Tools Required](#tools-required)
+- [Regex](#regex)
+- [What is JSON?](#what-is-json?)
+- [Linting](#linting)
+- [abstract, ident, and id](#abstract,-ident,-and-id)
+- [Weight and Volume](#weight-and-volume)
+- [Name](#name)
+- [Materials](#materials)
+- [Pocket Data](#pocket-data)
+- [Ammo Type](#ammo-type)
+- [Bleeding](#bleeding)
+- [barrel_length](#barrel_length)
+- [Weight and Volume](#weight-and-volume)
+- [copy-from and looks_like](#copy-from-and-looks_like)
+
+# Introduction
+Welcome to the Guide to updating legacy JSON. This is my first time using markdown or writing a guide, so please feel free to point out any errors or issues you see. Before you go any further, I highly reccommend you read the [Manual of Style](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md), the [Guide to adding new content to CDDA for first time contributors](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-contributors), and the [JSON Style Guide](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md). I created this guide due to an extreme lack of resources determining what, exactly was outtdated and had to be fixed. 
+
+# Tools Required
+A lot of this content requires find and replace, or at least find. To this end, I recommend you have a text editor with regex capabilities such as [Sublime Text 3](https://www.sublimetext.com/3) or [Notepad++](https://notepad-plus-plus.org/). I find Sublime the easiest to work with, but it's entirely your choice. Don't know what Regex is? Check out the section down below. 
+
```

Comment:
i use https://atom.io/ i find it much easier and ultra search - https://www.jam-software.com/ultrasearch, altho im going to give regex a try, can you add to the bottom goats recommendations for the mean time